explorer/e2e/testFunctions.ts

```diff
@@ -1,8 +1,22 @@
-import { expect, Page } from "@playwright/test";
+import { expect, Locator, Page } from "@playwright/test";
 import { TabDescription } from "./testInterfaces";
 
 /* eslint-disable sonarjs/no-duplicate-string  -- ignoring duplicate strings here */
 // Run the "Expect each tab to appear as selected when the corresponding url is accessed" test
+
+export const getFirstElementTextLocator = (
+  page: Page,
+  workColumnPosition: number
```

Comment:
Rename this to `columnIndex`? I'm not familiar with "work column" so feel free to keep it as is if this is a known concept.

Reply:
Nope, just a random variable name I thought of at the time but that only made sense in my head, I'll make sure to review those in more detail for future PRs!

Reply:
Can you go ahead and rename `workColumnPosition` to `columnIndex`, here and throughout?